octree/include/pcl/octree/octree_base.h

```diff
@@ -162,6 +165,23 @@ namespace pcl
           return BreadthFirstIterator (this, 0, NULL);
         };
 
+        // Octree breadth iterators at a given depth
+        typedef OctreeFixedDepthIterator<OctreeT> FixedDepthIterator;
+        typedef const OctreeFixedDepthIterator<OctreeT> ConstFixedDepthIterator;
+
+        FixedDepthIterator fixed_depth_begin (unsigned int fixed_depth_arg = 0u)
+        {
+          if (!(fixed_depth_arg <= this->octree_depth_))
+          {
+            PCL_WARN ("[pcl::octree::FixedDepthIterator] fixed_depth = %d (instead of %d)", this->octree_depth_, fixed_depth_arg);
```

Comment:
It's probably worth saying "The requested fixed depth was bigger than the octree's depth".